lib/block_producer.go

```diff
@@ -224,9 +224,7 @@ func (desoBlockProducer *DeSoBlockProducer) _getBlockTemplate(publicKey []byte)
 			if err != nil {
 				return nil, nil, nil, errors.Wrapf(err, "Error copying UtxoView: ")
 			}
-			_, _, _, _, err = utxoViewCopy._connectTransaction(
-				mempoolTx.Tx, mempoolTx.Hash, int64(mempoolTx.TxSizeBytes), uint32(blockRet.Header.Height), true,
-				false /*ignoreUtxos*/)
+			_, _, _, _, err = utxoViewCopy._connectTransaction(mempoolTx.Tx, mempoolTx.Hash, int64(mempoolTx.TxSizeBytes), uint32(blockRet.Header.Height), true, false)
```

Comment:
It would be nice if you could do a pass to fix all the line lengths that the refactoring broke. With vim you can highlight a line and hit "gq" and it does it. But not sure what the shortcut is in other things.